examples/gin-compat/handlers.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/go-fuego/fuego"
+)
+
+func ginController(c *gin.Context) {
+	c.String(200, "pong")
+}
+
+func fuegoControllerGet(c fuego.ContextNoBody) (HelloResponse, error) {
+	return HelloResponse{
+		Message: "Hello",
+	}, nil
+}
+
+func fuegoControllerPost(c fuego.ContextWithBody[HelloRequest]) (HelloResponse, error) {
+	body, err := c.Body()
+	if err != nil {
+		return HelloResponse{}, err
+	}
+
+	if body.Word == "forbidden" {
+		return HelloResponse{}, fuego.BadRequestError{Title: "Forbidden word"}
+	}
+
+	ctx := c.Context().(*gin.Context)
+	fmt.Printf("%#v", ctx)
```

Comment:
Left behind debug

Reply:
No it's to show usage :)

Reply:
Oh! Really?

I tend to use forbidigo to catch such left behind errors

Then I use nolintlint or simply fmt.Fprint* + os.Stout, for valid use cases, like yours